res/schema.xml

```diff
@@ -584,4 +584,50 @@ reapplying those migrations.
       UPDATE library SET filetype='aiff' WHERE filetype='aif';
     </sql>
   </revision>
+  <revision version="40" min_compatible="3">
+    <description>
+      Multi-Genre Support:
+      - Adds 'genres' table for centralized, hiererarchical genre management.
+      - Adds 'track_genres' junction table for many-to-many track-genre links.
+      - Migrates existing single genre-from 'library.genre' to the new tables.
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS genres (
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        name TEXT NOT NULL COLLATE NOCASE,
+        parent_id INTEGER DEFAULT NULL,
+        is_active INTEGER DEFAULT 1,
+        display_order INTEGER DEFAULT 0,
```

Comment:
"display_order" is associated to "by artist" or such. Is it a "sort_id"?

Reply:
Ah, it makes sens a simple number, populated as soon the user changes the order. The only issue is that the word order does not echo that 100% IMHO 

In this case, it is a number that is used for sorting. In case equal numbers a sorted by name. So I suggest to make this more clear with something like "sorting_number" "sort_key" or "display_order_index".